### ISSUE TITLE
Backward incompatible change in 5.1.5 and problem with sub dependencies in package.json

### ISSUE THREAD

**Issue by flekschas** [pixijs/pixi.js#6411]
This is the same issue as described in #6364 but for the `v5.1` branch.

It's not easily possible to downgrade from `v5.2` to `v5.1.5` because the sub-dependencies aren't fixed, e.g.,

```
"@pixi/accessibility": "^5.1.5",
```

which resolves to `5.2.1`

(See https://unpkg.com/browse/pixi.js@5.1.5/package.json)

Since #6364 is closed already and related to `v5.2` I am opening another issue. The trick with `"@pixi/*": "<5.2.0"` works but it's not obvious and requires yarn to work.

tl/dr

Why `v5.1`? Something in `v5.2` breaks the positioning of my scenegraph. Until I am able to figure out what's causing the issue in `v5.2` I would like to continue to use ` v5.1`.

**Comment by bigtimebuddy** [comment]
You could consider using [npm-force-resolutions](https://www.npmjs.com/package/npm-force-resolutions) if you aren't using yarn to provide similar functionality for resolving versions with npm. Very sorry for the inconvenience.

Also, we'd love to get to the bottom of your actual upgrading issue from 5.1 to 5.2. Please let us know when you find something there.

**Comment by flekschas** [comment]
Thanks for your quick reply! :)

Do you think it would be possible to release a patch for the `v5.1` branch with fixed sub-dependencies similar to `v5.2.1`?

I'll try to put together a small example for the `v5.2` I am experiencing and will open a separate issue for that.

**Comment by bigtimebuddy** [comment]
@flekschas I published v5.1.6 which is like 5.1.5 except all the dependencies are pinned like with 5.1.2. Hope that helps.

**Comment by flekschas** [comment]
Awesome! Thanks again for the superb response time and help. That fixes the issue.
